### PR TITLE
New aura filters

### PR DESCRIPTION
**Pull request by Phoenix-**
Is shown on global nameplates
Caster is player
Blizzard target frame default

### CODE REVIEW

File: Modules/Aura/FilterTypes.lua [hunk 1]
```diff
@@ -1323,6 +1323,127 @@ PitBull4_Aura:RegisterFilterType('Should consolidate',L["Personal nameplate"],pe
...
+	if (caster and (UnitIsUnit("player", caster) or UnitIsOwnerOrControllerOfUnit("player", caster))) then
+		return filter_result
+	end
+
+	if (UnitIsUnit("player", unit)) then
+		return filter_result
+	end
+
+	local casterIsAPlayer = entry[18]
+	local targetIsFriendly = not entry[23]
```
Comment (nebularg) [inline Modules/Aura/FilterTypes.lua]: Do the unit checks here and revert adding extra fields, please. Also rename the vars so they use snake_case 😜

File: Modules/Aura/FilterTypes.lua [hunk 2]
```diff
@@ -1323,6 +1323,127 @@ PitBull4_Aura:RegisterFilterType('Should consolidate',L["Personal nameplate"],pe
...
+				db.global_nameplate = false
+			end
+			PitBull4_Aura:UpdateAll()
+		end,
+		values = bool_values,
+		order = 1,
+	}
+end)
+
+-- Global nameplate aura, Filter by if the aura is eligible to show on all nameplates
```
Comment (nebularg) [inline Modules/Aura/FilterTypes.lua]: c/p oops

**Comment by Phoenix-** [comment]
I thought the entire aura entry is introduced to minimize Blizz API calls for performance reasons - so I went that way. If performance is not affected(API calls are fast enough) - surely extra elements are not required.
I'll fix it later today, thanks for your comments.

**Comment by nebularg** [comment]
The aura returns are cached so filters et al don't need to keep calling UnitAura (as an aside, I've been meaning to consolidate the update and highlight aura iterator forever). While I'm not super concerned about micro optimizations, ie, table lookup vs api call, I doubt that adding the unit checks for every aura are an efficiency improvement considering the filter can return early or not run at all.

**Comment by Phoenix-** [comment]
Reverted 3 new entry elements, left intact capturing of extra UnitAura return values.

**Comment by nebularg** [comment]
I implemented this using a meta filter in 1106f87, thanks for the input

**Comment by Phoenix-** [comment]
Nice, thanks.

**Pull request closed without merging**